### PR TITLE
WD-11514 Modified "Rock" to "rock" and linted the /ceph/install.html

### PR DESCRIPTION
**Pull request by immortalcodes**
## Done
- Changed occurance of "ROCKs" to "rocks" in ceph/install.html

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Also the [copydoc](https://docs.google.com/document/d/1EmOtHjuEMKhVLQA7GwyrKfCzOKfc8UCEyh-p04-PxzA/edit) for the change.
- [Demo](https://ubuntu-com-13892.demos.haus/ceph/install)

## Issue / Card
[WD-11514](https://warthogs.atlassian.net/browse/WD-11514)

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

[WD-11513]: https://warthogs.atlassian.net/browse/WD-11513?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[WD-11514]: https://warthogs.atlassian.net/browse/WD-11514?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: templates/ceph/install.html [hunk 1]
```diff
@@ -65,32 +65,29 @@ <h2>Single-node deployment</h2>
...
             </div>
           </div>
-          <hr class="p-separator is-shallow">
+          <hr class="p-separator is-shallow"/>
           <ol class="p-stepped-list">
             <li class="p-stepped-list__item">
-              <p class="p-stepped-list__title p-heading--4">
+              <p class="p-stepped-list__title p-heading--4"/>
                 To get started, install the MicroCeph snap with the following command on each node to be used in the cluster:
-              </h3>
```
[the remark this answers is not in the document]
Comment (immortalcodes) [reply templates/ceph/install.html:73]: I think if the `<p>` tag is followed by a `<div>` tag, closing it is not required

File: templates/ceph/install.html [hunk 2]
```diff
@@ -221,11 +210,11 @@ <h2>Multi-node deployment</h2>
         <section class="p-strip u-no-padding--top">
           <h2>Containerised deployment</h2>
           <ul class="p-list">
-            <li class="p-list__item is-ticked">Uses a Canonical-supplied and maintained ROCK (OCI image)</li>
+            <li class="p-list__item is-ticked">Uses a Canonical-supplied and maintained rock (OCI image)</li>
             <li class="p-list__item is-ticked">Works with cephadm and rook</li>
             <li class="p-list__item is-ticked">Suitable for all types of containerised deployments</li>
           </ul>
-          <p>These installation instructions use the Canonical produced and supplied Ceph ROCK &ndash; this OCI compliant image provides a drop in replacement for the upstream Ceph OCI image.</p>
+          <p>These installation instructions use the Canonical produced and supplied Ceph rock &ndash; this OCI compliant image provides a drop in replacement for the upstream Ceph OCI image.</p>
```
[the remark this answers is not in the document]
Comment (immortalcodes) [reply templates/ceph/install.html:217]: Yeah, you are right, it should not be `&ndash` but could it be hyphen?

File: templates/ceph/install.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (carkod) [reply templates/ceph/install.html:73]: Where is the source of this information? XHTML must always be closed by the same tag or self-closing as the W3C guidelines show.

File: templates/ceph/install.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (carkod) [reply templates/ceph/install.html:217]: Nope, hyphens are for compound words, modifiers, multiple words etc. [more here](https://www.grammarly.com/blog/hyphen/)

File: templates/ceph/install.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (immortalcodes) [reply templates/ceph/install.html:73]: image
I might not have interpreted this well

[the remark this answers is not in the document]
Comment (carkod) [reply templates/ceph/install.html:73]: Yeah but in this case you have a orphaned `</h3>`. Also, it helps readability, since there is a lot of text here.

**Review by carkod — APPROVED** [review]
Thanks for addressing the changes. LGTM!

**Pull request merged**